trillian_admin_api.proto

```diff
@@ -21,6 +21,7 @@ option java_outer_classname = "TrillianApiProto";
 package trillian;
 
 import "trillian.proto";
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
```

Comment:
It's also not consistent with other internal imports. The `crypto/sigpb/sigpb.proto` is imported as `github.com/google/trillian/crypto/sigpb/sigpb.proto`, for example. I think it's only protos that are in the same directory that are included using relative paths.